src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
```

Comment:
Creating an instance of `Application` to use `getRandomBall` method indicates that `getRandomBall` might be better off as a static method, or you should consider using an instance of `Lottery` instead of `Application` to call the method.

Reply:
getRandomBall is already static